pymc3/distributions/transforms.py

```diff
@@ -116,13 +119,16 @@ def __init__(self, a, b, eps=1e-6):
 
     def backward(self, x):
         a, b = self.a, self.b
-        r = (b - a) / (1 + tt.exp(-x)) + a
+        r = (b - a) * tt.nnet.sigmoid(x) + a
         return r
 
     def forward(self, x):
         a, b, e = self.a, self.b, self.eps
-        r = tt.log(tt.maximum((x - a) / tt.maximum(b - x, e), e))
-        return r
+        return tt.log(x - a) - tt.log(b - x)
```

Comment:
small thing, but there's a class variable `eps` that can be deleted (both here and in `__init__`.